docs/application/dotnet/guides/nui/xaml/introduction-with-example.md

```diff
@@ -0,0 +1,147 @@
+1. Create a XAML file
+2. Load the XAML file
+3. Put the XAML file in the appropriate file directory
+
+> **Note**
+> 
+> WPF XAML provides a template to create an application easily, but currently NUI XAML does not have a XAML application template. However, NUI XAML will be opened and released at **Tizen.NET API7** version.
+
+## Create XAML
+You can use XAML to easily implement the layout shown in the following figure:
+
+![AmbientPage](./Pictures/AmbientPage.PNG)
+
+Following is the XAML code:
+
+``` xaml
+<ContentPage x:Class="Tizen.NUI.Examples.AmbientPage"
+  xmlns="http://tizen.org/Tizen.NUI/2018/XAML"
+  xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml">
+
+  <ImageView Name="photo1" ResourceUrl="../../res/picture_07.jpg" Position2D="47,52" Size2D="440,440" />
+  <ImageView Name="photo2" ResourceUrl="../../res/picture_08.jpg" Position2D="47,512" Size2D="440,520" />
+  <ImageView Name="photo3" ResourceUrl="../../res/picture_02.jpg" Position2D="970,50" Size2D="440,560" />
+  <ImageView Name="photo4" ResourceUrl="../../res/picture_03.jpg" Position2D="970,630" Size2D="440,400" />
+  <ImageView Name="photo5" ResourceUrl="../../res/picture_04.jpg" Position2D="510,50" Size2D="440,280" />
+  <ImageView Name="photo6" ResourceUrl="../../res/picture_05.jpg" Position2D="510,350" Size2D="440,340" />
+  <ImageView Name="photo7" ResourceUrl="../../res/picture_06.jpg" Position2D="510,710" Size2D="440,320" />
+
+  <ImageView Name="photo0" ResourceUrl="../../res/bg_textbox_w.png" Position2D="1431,51" Size2D="440,600" />
+  <View Name="num_area" Size2D="440,160" Position2D="1431,51">
+    <ImageView Name="photo01" ResourceUrl="../../res/num_w_0.png" Position2D="0,0" Size2D="110,160" />
+    <ImageView Name="photo02" ResourceUrl="../../res/num_w_8.png" Position2D="110,0" Size2D="110,160" />
+    <ImageView Name="photo03" ResourceUrl="../../res/num_w_th.png" Position2D="220,0" Size2D="220,160" />
+  </View>
+  <View Name="mon_area" Size2D="440,194" Position2D="1431,211">
+    <ImageView Name="photo10" ResourceUrl="../../res/mon_w_4.png" Position2D="0,0" Size2D="440,194" />
+  </View>
+  <View Name="year_area" Size2D="440,210" Position2D="1431,425">
+    <ImageView Name="photo21" ResourceUrl="../../res/year_w_2.png" Position2D="0,0" Size2D="110,210" />
+    <ImageView Name="photo22" ResourceUrl="../../res/year_w_0.png" Position2D="110,0" Size2D="110,210" />
+    <ImageView Name="photo23" ResourceUrl="../../res/year_w_1.png" Position2D="220,0" Size2D="110,210" />
+    <ImageView Name="photo24" ResourceUrl="../../res/year_w_8.png" Position2D="330,0" Size2D="110,210" />
+  </View>
+  <ImageView Name="photo8" ResourceUrl="../../res/picture_01.jpg" Position2D="1431,671" Size2D="440,360" />
+</ContentPage>
+```
+
+In this example, the relative path of the image resources (relative to the location of the exe file) is used. You can also use the absolute path of the image.
+
+### Requirements:
```

Comment:
Please remove : at the end of headings

### Requirements

Reply:
I changed this as you had commented.